freeipa/src/main/java/com/sequenceiq/freeipa/sync/FreeipaChecker.java

```diff
@@ -82,6 +92,46 @@ public SyncResult getStatus(Stack stack, Set<InstanceMetaData> checkableInstance
         }
     }
 
+    private void checkNodeHealthReports(Stack stack, Set<InstanceMetaData> checkableInstances) {
+        for (InstanceMetaData instanceMetaData : checkableInstances) {
+            try {
+                LOGGER.debug("Fetching node health reports for instance: {}", instanceMetaData.getInstanceId());
+                RPCResponse<HealthReport> meteringReportRPCResponse = checkedMeasure(() -> freeIpaNodeStatusService.nodeMeteringReport(stack, instanceMetaData),
```

Comment:
it's true that we have a metering endpoint on every node, but it needs to be used only by datahub